MYSQL.md

```diff
@@ -144,5 +144,56 @@ $ mysql -u <username> -p
 ```
 
 
+## Linux
+
+Installation: 
+
+```Bash
+
+# If you do not have mysql-server installed run
+$ sudo pat-get install mysql-server
+# Enter a password you can rem when prompted during installation for root
+
+```
+Configure mapknitter account: 
+
+```Bash
+$ mysql -u root -p
+# Enter the password you input during installation
+# Create a mapknitter user
+$ CREATE USER 'mapknitter' IDENTIFIED BY 'password';
+# Grant permissions to the new user
+$ GRANT ALL ON *.* TO 'mapknitter' WITH GRANT OPTION;
+
+```
+
+_If you get the error `Access denied for user 'root'@'localhost'" error"`_  
+
+Meaning maybe you forgot your root password or was not prompted when installing. 
+You might be having some dependencies of mysql in your machine. You need to remove all the dependencies and reinstall mysql a fresh. 
+Use the following commands to remove mysql and all its dependencies
+
+```Bash
+$ sudo -i
+$ service mysql stop
+$ killall -KILL mysql mysqld_safe mysqld
+$ apt-get --yes purge mysql-server mysql-client
+$ apt-get --yes autoremove --purge
+$ apt-get autoclean
+$ deluser --remove-home mysql
+$ delgroup mysql
+$ rm -rf /etc/apparmor.d/abstractions/mysql /etc/apparmor.d/cache/usr.sbin.mysqld /etc/mysql /var/lib/mysql /var/log/mysql* /var/log/upstart/mysql.log* /var/run/mysqld
+$ updatedb
+$ exit
+
+````
+After this repeat installation and configuration steps
+
+
+Add the username and passsword on the `config/database.yml` development:  
```

Comment:
looks good to me! I'll try on my local machine. One thing that I found easier was to install mariadb instead of mysql... shouwl we also add a note just in case someone wants to use mariadb?

Reply:
@stefannibrasil I don't know do you think it's better off as a whole if everyone is using one DB even if installation might be harder for some individuals? There has been a lot of back and forth about this

Reply:
I'll always be for using just one database. I suggested this because mariadb is _almost_ the same as MySQL (for this case especifically), but yeah, I do think reinforcing just one db in the official doc is a good practice.

Reply:
@stefannibrasil Yeah I've read that too! I think the plots2 production database uses MariaDB but not sure